grafana/FilesystemDetails.json

```diff
@@ -2,7 +2,7 @@
   "__inputs": [
     {
       "name": "DS_PROMETHEUS",
-      "label": "PROMETHEUS",
+      "label": "Prometheus",
```

Comment:
I see that DS Label keeps changing between upper case and lower case. Please plan to keep it consistent.

Reply:
Any comment on DS Label ?